### PR TITLE
[#110557206 fixed] detach speech from speaker

### PR DESCRIPTION
**Pull request by vitalii-dmytruk**

### CODE REVIEW

File: src/main/java/com/intelliarts/conflab/core/repository/SpeechSpeakerRepository.java [hunk 1]
```diff
@@ -3,10 +3,12 @@
 import com.intelliarts.conflab.core.entity.Speaker;
 import com.intelliarts.conflab.core.entity.Speech;
 import com.intelliarts.conflab.core.entity.SpeechSpeaker;
+import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 import java.util.Set;
 
+@Repository
```
Comment (irudnyk) [inline src/main/java/com/intelliarts/conflab/core/repository/SpeechSpeakerRepository.java:11]: this annotation is extra here.

File: src/main/java/com/intelliarts/conflab/core/repository/EventSpeechSpeakerRepository.java [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+    @Modifying
+    @Query(value = "DELETE " +
+                   "FROM EventSpeechSpeaker eventSpeechSpeaker " +
+                   "WHERE eventSpeechSpeaker.event.id = :eventId " +
+                   "AND eventSpeechSpeaker.speechSpeaker.id IN (" +
+                   "    SELECT speechSpeaker.id " +
+                   "    FROM SpeechSpeaker speechSpeaker " +
+                   "    WHERE speechSpeaker.speaker.id = :speakerId" +
+                   ")")
+    void delete(@Param("speakerId") Long speakerId,@Param("eventId") Long eventId);
```
Comment (irudnyk) [inline src/main/java/com/intelliarts/conflab/core/repository/EventSpeechSpeakerRepository.java]: Please apply formatting to this file

File: src/main/java/com/intelliarts/conflab/core/repository/SpeechSpeakerRepository.java (hunk 1 above)
Comment (vitalii-dmytruk) [inline src/main/java/com/intelliarts/conflab/core/repository/SpeechSpeakerRepository.java:11]: Makes IDEA see this bean. Actually it does nothing more.

File: src/main/java/com/intelliarts/conflab/core/repository/EventSpeechSpeakerRepository.java (hunk 2 above)
Comment (vitalii-dmytruk) [inline src/main/java/com/intelliarts/conflab/core/repository/EventSpeechSpeakerRepository.java]: done

**Pull request merged**